boards/common/nrf52/Makefile.nrfmin.dep

```diff
@@ -1,5 +1,7 @@
-ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
-  ifeq (,$(filter nimble_% nrf802154,$(USEMODULE)))
-    USEMODULE += nrfmin
+ifneq (,$(filter netdev_default,$(USEMODULE)))
+  ifneq (,$(filter gnrc,$(USEMODULE)))
```

Comment:
`nrfmin` should work without `gnrc` so checking for `gnrc` here should not be necessary.